models/result.go

```diff
@@ -65,6 +66,7 @@ func (r *Result) HandleEmailSent() error {
 	r.SendDate = event.Time
 	r.Status = EventSent
 	r.ModifiedDate = event.Time
+	sendWebhooks(event)
```

Comment:
You should be able to remove all of these, since each `Handle*` function calls `r.createEvent` which [calls `campaign.AddEvent`](https://github.com/gophish/gophish/blob/52ed2eabe843033a03281a51b252c86d21092b57/models/result.go#L55). Since we're already sending the webhook in `AddEvent`, we'd wind up sending it twice as things are now.

Reply:
done